recipes/barrnap/0.2/meta.yaml

```diff
@@ -0,0 +1,22 @@
+about:
+  home: https://github.com/tseemann/barrnap
+  license: GPL3
+  summary: 'Barrnap predicts the location of 5S, 16S and 23S ribosomal RNA genes in
+    Bacterial genome sequ It takes FASTA DNA sequence as input, and write GFF3 as
+    output.'
+build:
+  number: 0
+package:
+  name: barrnap
+  version: '0.2'
+requirements:
+  build: []
```

Comment:
You can leave out above line if there are no build deps. Same for the other recipes.